voice/voice-ai/x/Actions/ActionHandler.swift

```diff
@@ -42,6 +43,14 @@ class ActionHandler: ObservableObject {
     func handle(actionType: ActionType) {
         switch actionType {
         case .reset:
+            tapCount += 1
+            
+            if tapCount % 7 == 0 {
+                DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
Should debounce to prevent rage click / accidental multiple clicks in short span of time

Reply:
I think this PR is no longer needed.

Reply:
Tap count % 7 might also be too many